src/tlo/methods/consumables.py

```diff
@@ -339,7 +339,7 @@ def check_format_of_consumables_file(df, fac_ids):
     months = set(range(1, 13))
     item_codes = set(df.item_code.unique())
 
-    assert set(df.columns) == {'Facility_ID', 'month', 'item_code', 'available_prop'}
+    assert set(df.columns) == {'Facility_ID', 'month', 'item_code', 'item_category', 'available_prop'}
```

Comment:
@tbhallett I have updated the `check_format_of_consumables_file` function to include a check for `item_category`. This refers to a cleaned categorisation of item_codes into programs or categories, such as "general" consumables. I use this columns to make regression based projections in PR #1367 as it's useful to have for costing plots. 
Is this OK?

Reply:
yes.